assembl/static2/yarn.lock

```diff
@@ -5,10 +5,12 @@
 "@2fd/command@^1.1.2":
   version "1.1.2"
   resolved "https://registry.yarnpkg.com/@2fd/command/-/command-1.1.2.tgz#b831d511bdd3b4e2d88139aaddb06d3d6aa0b08a"
+  integrity sha1-uDHVEb3TtOLYgTmq3bBtPWqgsIo=
```

Comment:
Could you please comment how you added the integrity field to yarn (along with why?).

Reply:
just execute yarn with latest version :)